pahfit/tests/test_classic_pack.py

```diff
@@ -17,9 +19,20 @@ def test_classic_pack():
                    sci_pack.att_info)
 
     # now read in the equivalent info from a file
-    path = pkg_resources.resource_filename('pahfit', 'packs/')
-    packfilename = '{}/scipack_ExGal_SpitzerIRSSLLL.ipac'.format(path)
-    pmodel = PAHFITBase(filename=packfilename, tformat='ipac')
+    path_packs = pkg_resources.resource_filename('pahfit', 'packs/')
+    packfilename = '{}/scipack_ExGal_SpitzerIRSSLLL.ipac'.format(path_packs)
+    path_data = pkg_resources.resource_filename('pahfit', 'data/')
+    datafilename = '{}/M101_Nucleus_irs.ipac'.format(path_data)
+
+    obs_spectrum = Table.read(datafilename, format='ipac')
+    obs_x = obs_spectrum['wavelength'].to(u.micron,
+                                          equivalencies=u.spectral())
+    obs_y = obs_spectrum['flux'].to(u.Jy,
+                                    equivalencies=u.spectral_density(obs_x))
+    obs_x = obs_x.value
+    obs_y = obs_y.value
+
+    pmodel = PAHFITBase(obs_x, obs_y, filename=packfilename, tformat='ipac')
```

Comment:
Why not just use obs_x.value and obs_y.value in the call to PAHFITBase?  Then you don't need lines 32-33.